### PR TITLE
move %install_info to file triggers (boo#1152105)

### PR DESCRIPTION
**Pull request by lnussel**

### CODE REVIEW

**Comment by lnussel** [comment]
ping!

File: suse_macros.in [hunk 1]
```diff
@@ -222,18 +222,11 @@
 
 # for %post
 %install_info(:-:) \
-    if test -x /sbin/install-info ; then \
-	/sbin/install-info %{**} || : \
-    fi ;
+   # legacy install_info macro used, please fix the package
```
Comment (Conan-Kudo) [inline suse_macros.in]: Wouldn't it be better to use `%{error:}` or `%{warn:}` here? https://rpm.org/user_doc/macros.html#builtin-macros

Comment (lnussel) [reply suse_macros.in]: I was actually thinking of writing an rpmlint check that triggers on that comment in binary rpms. %warn wouldn't leave traces in the binary rpm so nothing we can look for.
Grepping the spec file on the other hand would be too naive as the macro could be wrapped in %if ...

Comment (Conan-Kudo) [reply suse_macros.in]: Using `%error` would stop the package from being able to use the macro when it's not supposed to be used.

Comment (lnussel) [reply suse_macros.in]: that's not what we normally do to deprecate macros. Would force packagers to wrap in %if %suse_version themselves which is annoying.

Comment (lnussel) [reply suse_macros.in]: I'd rather expand to %{nil} and forget about checks. spec_cleaner could remove it then.

Comment (Conan-Kudo) [reply suse_macros.in]: That works too.

**Pull request merged**